lib/Client/Util.php

```diff
@@ -49,4 +62,30 @@ public function pushWithMax(&$arr, $item, $max) {
             return false;
         }
     }
+
+    public static function hexdec($input) {
+        $str_high = substr($input, 0, 8);
+        $str_low = substr($input, 8, 8);
```

Comment:
it looks like we are creating strings that are longer than 9 places. is this cutting short or purpose?

`public` function _generateUUIDString() {
        // must return less than 7fffffffffffffff`

Reply:
I'm certain this is related to everything you talked about, so this is probably intentional, I guess some comments would be helpful.

Reply:
also probably not, but would http://php.net/manual/en/function.unpack.php be helpful at all?

Reply:
I put a link to the resource I got this from.